src/printer.js

```diff
@@ -1720,7 +1720,8 @@ function printArgumentsList(path, options, print) {
     lastArg.type === "FunctionExpression" ||
     lastArg.type === "ArrowFunctionExpression" &&
       (lastArg.body.type === "BlockStatement" ||
-        lastArg.body.type === "ArrowFunctionExpression") ||
+        lastArg.body.type === "ArrowFunctionExpression" ||
```

Comment:
Is there a reason why we have to enumerate all the types of body we want to support? Could I just get rid of the check on body.type?